clients/fides-js/src/lib/initialize.ts

```diff
@@ -131,22 +131,34 @@ const automaticallyApplyGPCPreferences = ({
         gpcApplied = true;
         return new SaveConsentPreference(
           notice,
-          transformConsentToFidesUserPreference(false, notice.consent_mechanism)
+          transformConsentToFidesUserPreference(
+            false,
+            notice.consent_mechanism
+          ),
+          // TODO (PROD-1597): ...figure out the right translation for these preferences, somehow?
```

Comment:
TODO: This is the major outstanding item on this PR; how we handle GPC notices needs to be carved out into a separate PR, so this just needs a bit of tidying to be _stable_ 👍

Reply:
Done!